### PR TITLE
Improve radio button

### PR DESCRIPTION
**Pull request by angsherpa232**

### CODE REVIEW

File: src/components/Recommendation.js [hunk 1]
```diff
@@ -3,14 +3,14 @@ import PropTypes from 'prop-types';
...
-import { Grid, Header, List, Tab, Form } from 'semantic-ui-react';
+import { Grid, Header, List, Tab, Button } from 'semantic-ui-react';
 
 import { ReactComponent as NegativeIcon } from '../icons/recommendationNegative.svg';
 import { ReactComponent as NeutralIcon } from '../icons/recommendationNeutral.svg';
 import { ReactComponent as PositiveIcon } from '../icons/recommendationPositive.svg';
 import styles from './Recommendation.module.css';
 
-function Recommendation({ todayFutureToggler }) {
+function Recommendation({ todayFutureToggler, disabled }) {
```
Comment (friedjoff) [inline src/components/Recommendation.js]: Do we still need `todayFutureToggler`?

File: src/components/Recommendation.js [hunk 2]
```diff
@@ -48,24 +48,26 @@ function Recommendation({ todayFutureToggler }) {
...
-            </Form.Field>
-          </Form>
+          <Button.Group>
+            <Button
+              disabled={disabled}
+              active={!future}
+              onClick={() => setFuture(false)}
+              className={styles.button}
+            >
+              <h5>Heute</h5>
```
Comment (friedjoff) [inline src/components/Recommendation.js]: Please use the translate function.

File: src/components/Recommendation.js (hunk 1 above)
Comment (angsherpa232) [reply src/components/Recommendation.js]: removed.

File: src/components/Recommendation.js (hunk 2 above)
Comment (angsherpa232) [reply src/components/Recommendation.js]: Done.

File: src/components/Recommendation.js [hunk 3]
```diff
@@ -3,14 +3,14 @@ import PropTypes from 'prop-types';
...
-import { Grid, Header, List, Tab, Form } from 'semantic-ui-react';
+import { Grid, Header, List, Tab, Button } from 'semantic-ui-react';
 
 import { ReactComponent as NegativeIcon } from '../icons/recommendationNegative.svg';
 import { ReactComponent as NeutralIcon } from '../icons/recommendationNeutral.svg';
 import { ReactComponent as PositiveIcon } from '../icons/recommendationPositive.svg';
 import styles from './Recommendation.module.css';
 
-function Recommendation({ todayFutureToggler }) {
+function Recommendation({ disabled }) {
```
Comment (friedjoff) [inline src/components/Recommendation.js]:
```suggestion
function Recommendation({ futureDisabled }) {
```

File: src/components/Recommendation.module.css [hunk 4]
```diff
@@ -3,3 +3,30 @@
...
+  flex: auto !important;
+  color: var(--primary-color) !important;
+  background: #fff;
+}
+
+.button:global(.ui.active.button) {
+  background: #f2f4f2;
+}
+
+.button:global(.ui.active.button:hover) {
```
Comment (friedjoff) [inline src/components/Recommendation.module.css]: It seems this and the next style can be combined.

File: src/components/Recommendation.module.css [hunk 5]
```diff
@@ -3,3 +3,30 @@
...
+.button:global(.ui.button:hover) {
+  background: #fff;
+}
+
+.button h5 {
+  margin-bottom: 5px;
+}
+
+.button p {
+  line-height: 1rem;
```
Comment (friedjoff) [inline src/components/Recommendation.module.css]:
```suggestion
```

File: src/components/Recommendation.js [hunk 6]
```diff
@@ -47,26 +47,28 @@ function Recommendation({ todayFutureToggler }) {
...
-        )}
+
+        <Button.Group>
+          <Button
+            disabled={disabled}
+            active={!future}
+            onClick={() => setFuture(false)}
+            className={styles.button}
+          >
+            <h5>{t('recommendation.today')}</h5>
```
Comment (friedjoff) [inline src/components/Recommendation.js]:
```suggestion
            <h5>{t('recommendation.todayHeader')}</h5>
```

File: src/components/Recommendation.js [hunk 7]
```diff
@@ -47,26 +47,28 @@ function Recommendation({ todayFutureToggler }) {
...
+            <p>{t('todayFutureToggler.today')}</p>
```
Comment (friedjoff) [inline src/components/Recommendation.js]:
```suggestion
            <p>{t('recommendation.todayMessage')}</p>
```

File: src/components/Recommendation.module.css [hunk 8]
```diff
@@ -3,3 +3,30 @@
...
 }
+
+.button:global(.ui.button) {
+  flex: auto !important;
+  color: var(--primary-color) !important;
+  background: #fff;
+}
+
+.button:global(.ui.active.button) {
+  background: #f2f4f2;
```
Comment (friedjoff) [inline src/components/Recommendation.module.css]:
```suggestion
  background: #f2f2f2;
```

File: src/components/Recommendation.module.css [hunk 9]
```diff
@@ -3,3 +3,30 @@
...
+}
+
+.button:global(.ui.active.button:hover) {
+  background: #fff;
+}
+
+.button:global(.ui.button:hover) {
+  background: #fff;
```
Comment (friedjoff) [inline src/components/Recommendation.module.css]:
```suggestion
  background: #f7f7f7;
```

Comment (angsherpa232) [reply src/components/Recommendation.module.css]: This brings a visible change in the color, does not seems to be constant with other buttons.

**Pull request merged**